sdk/storage/azure-storage-file-datalake/tests/test_quick_query.py

```diff
@@ -866,4 +866,47 @@ def on_error(error):
                 on_error=on_error,
                 file_format=input_format)
 
+    @DataLakePreparer()
+    def test_quick_query_input_in_parquet_format(self, datalake_storage_account_name, datalake_storage_account_key):
+        # Arrange
+        self._setUp(datalake_storage_account_name, datalake_storage_account_key)
+        file_name = self._get_file_reference()
+        file_client = self.dsc.get_file_client(self.filesystem_name, file_name)
+        errors = []
+
+        def on_error(error):
+            errors.append(error)
+        expression = "select * from blobstorage where id < 1;"
+        expected_data = b"0,mdifjt55.ea3,mdifjt55.ea3\n"
+
+        parquet_path = os.path.abspath(os.path.join(os.path.abspath(__file__), "..", "./resources/parquet.parquet"))
+        with open(parquet_path, "rb") as parquet_data:
+            file_client.upload_data(parquet_data, overwrite=True)
+
+        reader = file_client.query_file(expression, file_format=QuickQueryDialect.ParquetDialect, on_error=on_error)
+        real_data = reader.readall()
+
+        self.assertEqual(real_data, expected_data)
+
+    @DataLakePreparer()
+    def test_quick_query_output_in_parquet_format(self, datalake_storage_account_name, datalake_storage_account_key):
+        # Arrange
+        self._setUp(datalake_storage_account_name, datalake_storage_account_key)
+        file_name = self._get_file_reference()
+        file_client = self.dsc.get_file_client(self.filesystem_name, file_name)
+        errors = []
+
+        def on_error(error):
+            errors.append(error)
```

Comment:
these are not being used since there's no place checking if there were error during service processing the content

Reply:
Yes my bad I forgot to remove these two lines! good catch